packages/firebase_performance/firebase_performance/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.7.1
+- **FEAT**: add web support.
```

Comment:
please remove changes to changelog. Our repo orchestration tool ([melos](https://github.com/invertase/melos)) handles changelog updates from commits/pr titles.